AdminWebsite/AdminWebsite.AcceptanceTests/AdminWebsite.AcceptanceTests.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="BookingsApi.Client" Version="1.27.8" />
+    <PackageReference Include="BookingsApi.Client" Version="1.27.9-pullrequest0321-0003" />
```

Comment:
Is this intentional? should not we use a steady version 1.27.9 or 1.27.10 instead?